api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,10 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="Add Category field to OrderSet" author="odorajonathan">
```

Comment:
Can you look at the changesets in this file to see how yours should be? liquibase-update-to-2.0.xml

Reply:
OK! I hope when u say liquibase-update-to-2.0.xml u mean I should change the liquibase changeset for the field to that file other than the liquibase-update-to-2.3.xml I originally put it in.

Reply:
NO. I mean look at those changesets for examples of how your changeset should be.

Reply:
Hey @dkayiwa is the updated code change viable? I still see failures, what am I missing to add here. Please review

Reply:
Did you look at the error log?

Reply:
Yea, I have been looking at it but I don't get it that well. Keeps referring to `transactionManager` something reporting error from ` org.springframework.beans.factory.BeanCreationException`, I don't know if that means there's what am referencing to or creating that is not recognized throwing the beanCreationException

Reply:
Did you even try to ask on talk?